pkg/harvester/models/harvesterhci.io.virtualmachinebackup.js

```diff
@@ -81,23 +80,22 @@ export default class HciVmBackup extends HarvesterResource {
   }
 
   restoreExistingVM(resource = this) {
-    const route = this.currentRoute();
     const router = this.currentRouter();
 
     router.push({
       name:   `${ HARVESTER_PRODUCT }-c-cluster-resource-create`,
-      params: { resource: route.params.resource },
+      params: { resource: HCI.BACKUP },
```

Comment:
I think this is common with Snapshots functionalities, so I guess this is causing the wrong route error when clicking restore in Snapshots page.

Reply:
fix in eabd9435